src/commands/txCommands/util/gitHelpers.js

```diff
@@ -21,12 +22,49 @@ const commit$ = (commitMessage, args) => {
 			child_process.execSync('git add .');
 			const command = `git commit -m ${JSON.stringify(commitMessage)} ${args}`;
 			return child_process$(command)
-			.do(() => {
-				console.log(chalk.green(command));
-			});
+				.do(() => {
+					console.log(chalk.green(command));
+				});
 		});
 };
 
+const devGitBranch$ = Rx.Observable.bindNodeCallback(child_process.exec)(
```

Comment:
the 'bound' version of `child_process.exec` has already been assigned to a const you can re-use here
```js
const devGitBranch$ = child_process$(...)
```